features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,330 @@
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2016-10-01          |
+    | datumTot            | 2020-07-01          |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat      | datumAanvangAdreshouding | datumAanvangVolgendeAdreshouding |
+    | 0800010000000002                 | Luttestraat | 20160526                 | 20201008                         |
+    # 24 |--A1--|--A2--|--A3--
+    #            |---|
+
+  Scenario: De gevraagde periode ligt tussen datum aanvang adreshouding op een locatie en datum aanvang volgende adreshouding
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A4' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op adres 'A3' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20201008                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2016-10-01          |
+    | datumTot            | 2020-07-01          |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | locatiebeschrijving         | datumAanvangAdreshouding | datumAanvangVolgendeAdreshouding |
+    | Woonboot bij de Grote Sloot | 20100818                 | 20201008                         |
+
+  Scenario: De gevraagde periode ligt na datum aanvang adres buitenland
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op een buitenlands adres met de volgende gegevens
+    | datum aanvang adres buitenland (13.20) | land adres buitenland (13.10) |
+    | 20181201                               | 6014                          |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2019-10-01          |
+    | datumTot            | 2020-07-01          |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | land.code | land.omschrijving            | datumAanvangAdreshouding | datumAanvangVolgendeAdresBuitenland |
+    | 6014      | Verenigde Staten van Amerika | 20181201                 |                                     |
```

Comment:
de datum aanvang adres buitenland moet niet geleverd worden in datumAanvangAdreshouding, maar in datumAanvangAdresBuitenland

Reply:
aangepast